libindy/src/domain/anoncreds/credential_attr_tag_policy.rs

```diff
@@ -0,0 +1,46 @@
+use std::collections::HashSet;
+
+use serde::ser::{Serialize, Serializer, SerializeSeq};
+use serde::de::{Deserializer, Deserialize};
+
+use named_type::NamedType;
+
+fn canon(attr: &str) -> String {
```

Comment:
Are we really would like to have this function? I was thinking that CredentialAttrTagPolicy is used only from few places and in all of them we are operating by full, non-modified name.